player_alsa.h

```diff
@@ -0,0 +1,33 @@
+#include <alsa/asoundlib.h>
+
+void check(int *err, int newErr) {
```

Comment:
The file is #included in the Go file :D. The reason why I put it in a separate file is to get syntax highlighting and compilation errors.